.github/workflows/ci.yaml

```diff
@@ -0,0 +1,134 @@
+# Add a comment to force a change.
+
+name: Webhook CI
+
+on:
+  workflow_call:
+  push:
+    branches:
+      - release/v*
+    paths-ignore:
+      - '*.md'
+      - '.gitignore'
+      - 'CODEOWNERS'
+      - 'LICENSE'
+  pull_request:
+    paths-ignore:
+    - '*.md'
+    - '.gitignore'
+    - 'CODEOWNERS'
+    - 'LICENSE'
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+    name: CI
+    runs-on : ubuntu-latest
+    strategy:
+      matrix:
+          arch:
+          - amd64
+          - arm64
+    steps:
+      - name : Checkout repository
+        uses : actions/checkout@v4
+        with :
+          fetch-depth : 0
+      - run : make ci
+        name: ci
+
+      - name : Save builds
+        run : |
+          mkdir -p dist/artifacts
+          cp bin/* dist/artifacts/
+      - name: Upload artifact
+        uses: actions/upload-artifact@v4
+        with:
+          name: webhook-build-artifacts-${{ matrix.arch }}
+          path: ./dist/artifacts
+
+  test:
+    name : integration-test
+    needs : [
+      build
+    ]
+    runs-on : ubuntu-latest
+    strategy:
+      matrix:
+        arch:
+        - amd64
+        - arm64
+    steps:
+      - name : Checkout repository
+        uses : actions/checkout@v4
+        with :
+          fetch-depth : 0
+
+      # TODO: Pull this next one out once there's a release of rancher 2.9 that runs on k8s 1.29
+      - name : Checkout rancher/rancher and build the chart
+        run: |
+          pushd "${{ runner.temp}}"
+          git clone https://github.com/rancher/rancher.git rancherDir
+          cd rancherDir
+          git checkout release/v2.9
+          ./scripts/chart/build chart
+          sed -i -e 's/kubeVersion: < 1.29.0-0/kubeVersion: < 1.30.0-0/' build/chart/rancher/Chart.yaml
+          tar cfz "${{ runner.temp }}/rancher.tgz" -C build/chart/rancher .
+          popd
+
+      - name : Fetch build artifacts
+        uses: actions/download-artifact@v4
+        with:
+          name: webhook-build-artifacts-${{ matrix.arch }}
+          path: ./dist/artifacts
+
+      - name : Debug
+        run : ls -R ./dist/artifacts
+
```

Comment:
Is this meant to be kept here?

Reply:
Nope, that's how I discoverd `cp`ing from `dist/artifacts` was losing the `x` bits